Library/Formula/gcc.rb

```diff
@@ -61,8 +61,8 @@ def osmajor
 
   # The bottles are built on systems with the CLT installed, and do not work
   # out of the box on Xcode-only systems due to an incorrect sysroot.
-  def pour_bottle?
-    MacOS::CLT.installed?
+  pour_bottle? "The bottle needs the Xcode CLT to be installed." do
```

Comment:
I find “Xcode CLT” to be massively confusing. Apple refers to them as “Command Line Tools for Xcode” and we usually just refer as “CLT” to them. Can we make that “The bottle needs the Command Line Tools to be installed.” instead (here and in all other formulae with the same condition)?